internal/appstore/appstore.go

```diff
@@ -197,7 +189,7 @@ func calculateBundleHash(filename string) (string, error) {
 }
 
 func createLocateRequest(url, username, accesskey string, queryString string) (*http.Request, error) {
-	req, err := requesth.New(http.MethodGet, fmt.Sprintf("%s%s&per_page=100", url, queryString), nil)
+	req, err := requesth.New(http.MethodGet, fmt.Sprintf("%s%s&per_page=1", url, queryString), nil)
```

Comment:
Shouldn't the parameter be the hash instead of a partial query string ?
(So actual request is abstracted)

It's ok even without that change 👍

Reply:
yep. exactly :D